db-tools-i18n/src/main/resources/com/splicemachine/db/loc/messages.xml

```diff
@@ -290,12 +290,12 @@ Guide.
                 <text>A connection could not be established because the password has a length of zero or is larger than the maximum allowed by the network protocol.</text>
             </msg>
 
-			<msg>
+            <msg>
```

Comment:
I wouldn't change white spaces if possible to keep Git history clean, maybe dedicate a JIRA for that effort instead.

Reply:
fyi, you can filter whitespace changes when displaying git history.

Reply:
Sure, but I am not sure you can do that in `git blame` or `git log` for example.